### PR TITLE
Added intitial docs for developing collectors

### PR DESCRIPTION
**Pull request by gesparza3**
This PR adds documentation for developing collectors for Rode. Topics discussed are:
  - Role of collectors in Automated Governance
  - Responsibilities each collector should take on
  - Approaches for aggregating metadata
  - Resources for getting started on your own collector

### CODE REVIEW

**Review by RobertKelly — APPROVED** [review]
Great add on collectors.

File: docs/collectors.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+and add new functionality at their own pace.
+
+## Development
+
+As mentioned previously, collectors are independent services and can be built with
+any language or framework that has support for gRPC. Collectors typically have two
+primary responsibilities:
+
+- Listening for incoming event metadata associated with artifacts _for a single service_
+- Publishing structured data to the Rode API in the form of _occurrences_
```
Comment (aalsabag) [inline docs/collectors.md:20]: Should we indicate anywhere that you can also go back and retrieve more info if you want

**Review by aalsabag — APPROVED** [review]

File: docs/collectors.md (hunk 1 above)
Comment (gesparza3) [reply docs/collectors.md:20]: Good idea @aalsabag, just added a bit more info. Let me know what you think

Comment (aalsabag) [reply docs/collectors.md:20]: I like that last paragraph. Nice job @gesparza3 !

**Pull request merged**